crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -3214,6 +3214,19 @@ where
                     flake8_bandit::rules::hardcoded_sql_expression(self, expr);
                 }
             }
+            ExprKind::BinOp {
+                op: Operator::BitOr,
+                ..
+            } => {
+                if self.is_stub {
```

Comment:
Oh sorry -- I also changed this to `.pyi` only for consistency with the other `flake8-pyi` rules. I want to enable these for non-`.pyi` files, but I want to do it all at once for the plugin.